src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -470,10 +470,10 @@
                 </select>
                 <select id="revision-select" aria-label="Revision Select">
                     @{
-                        var revisionsToSelectFrom = Model.ReviewContent.APIRevisions[Model.ReviewContent.ActiveRevision.APIRevisionType.ToString()];
+                        var revisionsToSelectFrom = Model.ReviewContent.APIRevisionsGrouped[Model.ReviewContent.ActiveAPIRevision.APIRevisionType.ToString()].OrderByDescending(c => c.CreatedOn);

```

Comment:
curious, what type is `APIRevisionType` that always needs to be converted to String?

Reply:
Its an Enum. It used to be `ReviewType`